pyuvdata/uvdata/miriad.py

```diff
@@ -772,6 +779,7 @@ def write_miriad(
                 check_extra=check_extra,
                 run_check_acceptability=run_check_acceptability,
                 check_freq_spacing=True,
+                uvw_antpos_check_level="off",
```

Comment:
I'm not sure. My thinking was that the check takes some time and we might not want to do it on both read and write. But maybe I shouldn't be so conservative about that. I guess if someone makes an object directly rather than reading a file the check wouldn't get run by default, which might be bad.

Reply:
We decided to always have it on if `run_check_acceptability` is True, and to use a new keyword, `strict_uvw_antpos_check` to allow it to error rather than just issuing a warning.